kratos/utilities/nurbs_utilities/projection_nurbs_geometry_utilities.h

```diff
@@ -95,9 +95,9 @@ class ProjectionNurbsGeometryUtilities
             if (norm_2(delta_t * derivatives[1]) < Accuracy)
                 return true;
 
-            // Check if the parameter gets out of its interval of definition and if so clamp it 
+            // Check if the parameter gets out of its interval of definition and if so clamp it
             // back to the boundaries
-            int check = rGeometry.ClosestPointLocalToLocalSpace(rParameterLocalCoordinates);
+            int check = rGeometry.ClosestPointLocalToLocalSpace(rParameterLocalCoordinates, rParameterLocalCoordinates);
```

Comment:
are you sure that this is not a bit evil?

Reply:
I think there shouldn't be any problem (note that the first reference is taken as `const`). I think that doing a copy with the unique purpose of passing it to the `ClosestPointLocalToLocalSpace` wasn't very good idea...